soc/arm/st_stm32/stm32wb/Kconfig.defconfig.stm32wb55xx

```diff
@@ -11,8 +11,4 @@ config SOC
 config NUM_IRQS
 	default 63
 
-config DMAMUX_STM32
```

Comment:
removing the dmamux for **wb** soc because the  _drivers/dma/Kconfig.stm32_ enables for this serie.
--> change the PR title or add another PR.

Reply:
else LGTM

Reply:
@FRASTM done